linux_os/guide/system/permissions/files/dir_perms_world_writable_system_groupowned/rule.yml

```diff
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+prodtype: sle12
+
+title: 'Ensure All World-Writable Directories Are Group-Owned by a System Group'
```

Comment:
This rule looks very much like this one: https://github.com/ComplianceAsCode/content/tree/master/linux_os/guide/system/permissions/files/dir_perms_world_writable_system_owned_group

Rule description is a bit different, but behavior seems the same.
Could you highlight the difference? Maybe I missed a detail.

Reply:
Good catch! Re-reading the descriptions for SLES-12-010830 and RHEL-07-021030, they seem to mean the same thing! I'll update the rule.